### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by ghost** [cSploit/android#904]
There have not been any commits for 4 months now. Is this dead?

**Comment by pahapoika** [comment]
Hey @boomtnt46
The development is far from dead. cSploit V2 is currently on progress and alpha build should be available soon.

**Comment by thezakman** [comment]
@pahapoika what does "soon" means?

**Comment by pahapoika** [comment]
@thezakman anything from day to 3-months

**Comment by p4cm4n** [comment]
> what does "soon" means?

https://developer.valvesoftware.com/wiki/Valve_Time

**Comment by ghost** [comment]
Yeah, this project is dead. I was hoping cSploit was gonna blow everyone away, but with all these problems and networks getting smarter to these attacks and more adept IT, you can hardly do anything without getting caught with these apps.

**Comment by keeganjk** [comment]
Still waiting for cSploit v2...

**Comment by pahapoika** [comment]
[Please see my reply regarding cSploit 2](https://github.com/cSploit/android/issues/924#issuecomment-355210423)

**Comment by fat-tire** [comment]
And mine in #938

**Comment by keeganjk** [comment]
It's literally been almost half a year since this issue was opened. Will it *ever* be finished? (Sorry for being impatient.)

**Comment by ETeissonniere** [comment]
Hey guys, one of the devs/contributor here working on the new version, please note we are working **during our free time** on this project, as everyone has a *life*, is *studying* and *working to be able to live* we can't work on it full time.

So saying the project is dead and *insisting on that every month* is a bit hard from you guys, one of the rules in an open source community is **don't ask for an ETA**, if you want things to go faster, feel free to submit pull requests or finance us.

As for the "getting caught" part I would like to raise that "this kind of apps" if not designed for **script kiddies** who just thinks they can press a button and hack the NASA, it is intended *for people who know what they are doing* and mostly professionals or passionates.

As of now, I will refer any other requests asking for an ETA to this reply.

**Comment by p4cm4n** [comment]
[So we cannot do this?](https://media.giphy.com/media/eCqFYAVjjDksg/giphy.gif)

Now seriously, thanks for the hard work, this app is one of a kind :)

**Comment by keeganjk** [comment]
@DeveloppSoft Do you have like a... *List* of things you need help on and which files the changes would occur in, etc.?

**Comment by CoffeeNData** [comment]
Really soon

**Comment by thezakman** [comment]
This is dead.

**Comment by Dawserdoos** [comment]
I believe it is as well... Too bad...
